test/e2e/create_test.go

```diff
@@ -108,15 +108,15 @@ func (tc *testContext) waitForWindowsNodes(nodeCount int32, waitForAnnotations,
 	if expectError {
 		// The time we expect to wait, if the windowsLabel is
 		// not used while creating nodes.
-		creationTime = time.Minute * 5
+		creationTime = time.Minute * 10
```

Comment:
Is this correct fix? This increases the time we will wait if we create a Machine without the `windowsLabel`. What does it have to do with deletion?